app/models/solution.rb

```diff
@@ -1,4 +1,13 @@
 class Solution < ApplicationRecord
   belongs_to :question
-  validates :content, presence: true
+  has_many :favorites
+
+  # validates :content, presence: true
+  with_options presence: true do
+    validates :content
+  end
+
+  def favorited_by?(user)
```

Comment:
```suggestion
  def favorited_by?(user_id)
```
少し細かいですが、メソッド内でidか使わないのであれば、idを引数で渡し、実装するのが良いと思います！